### PR TITLE
feature-erms-1363

### PR DESCRIPTION
**Pull request by croqueGrec09**
as of ERMS-1363, orgs can have now notes and tasks like subscriptions or licenses

### CODE REVIEW

File: app/grails-app/controllers/de/laser/OrganisationController.groovy [hunk 1]
```diff
@@ -801,6 +809,42 @@ class OrganisationController extends AbstractDebugController {
...
+                }
+                catch (Exception e) {
+                    flash.message = message(code: 'default.not.deleted.message', args: [message(code: 'task.label'), params.deleteId])
+                }
+            }
+        }
+
+        int offset = params.offset ? Integer.parseInt(params.offset) : 0
+        result.taskInstanceList = taskService.getTasksByResponsiblesAndObject(result.user, result.institution, result.orgInstance)
+        result.taskInstanceCount = result.taskInstanceList?.size()
```
Comment (klodav) [inline app/grails-app/controllers/de/laser/OrganisationController.groovy:837]: taskService.getTasksByX() always returns (empty) lists

File: app/grails-app/controllers/de/laser/OrganisationController.groovy [hunk 2]
```diff
@@ -801,6 +809,42 @@ class OrganisationController extends AbstractDebugController {
...
+        result.taskInstanceList = taskService.chopOffForPageSize(result.taskInstanceList, result.user, offset)
+
+        result.myTaskInstanceList = taskService.getTasksByCreatorAndObject(result.user,  result.orgInstance)
+        result.myTaskInstanceCount = result.myTaskInstanceList?.size()
```
Comment (klodav) [inline app/grails-app/controllers/de/laser/OrganisationController.groovy:841]: taskService.getTasksByX() always returns (empty) lists

Comment (croqueGrec09) [reply app/grails-app/controllers/de/laser/OrganisationController.groovy:841]: Okay, the ? will be removed.

File: app/grails-app/controllers/de/laser/OrganisationController.groovy (hunk 1 above)
Comment (croqueGrec09) [reply app/grails-app/controllers/de/laser/OrganisationController.groovy]: Removed in 3b2b547

File: app/grails-app/controllers/de/laser/OrganisationController.groovy (hunk 2 above)
Comment (croqueGrec09) [reply app/grails-app/controllers/de/laser/OrganisationController.groovy]: Removed in 3b2b547

**Review by klodav — APPROVED** [review]

**Pull request merged**